crosschain/simple/contracts/Sc1Contract1.sol

```diff
@@ -0,0 +1,19 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+pragma solidity ^0.5;
+
+import "./Crosschain.sol";
+import "./Sc2Contract2Interface.sol";
+
+contract Sc1Contract1 is Crosschain {
+    uint256 private sc2;
+    Sc2Contract2Interface private contract2;
+
+    constructor (uint256 _Sc2SidechainId, address payable _contract2 ) public {
```

Comment:
I don't think _contract2 should be  payable

Reply:
Fixed